packages/cactus-plugin-ledger-connector-corda/src/main/typescript/plugin-ledger-connector-corda.ts

```diff
@@ -146,6 +151,14 @@ export class PluginLedgerConnectorCorda
     return webServices;
   }
 
+  private readPrivateKey(): void {
```

Comment:
@elenaizaguirre Please make the name a little more descriptive. At first glance it might not be clear what kind of private key the method is operating with so a quick sentence for the documentation would be cool and maybe a slighly longer name like `readSshPrivateKey()` or something.

Reply:
Yes, we should be avoiding general names like `privateKey` , `publicPrivatekeyPair`, and methods related to that etc, and replace them with a bit more understandable naming schema like the one @petermetz mentioned (`sshPrivateKey`)

Reply:
Done